docs/get_started.rst

```diff
@@ -1,17 +1,341 @@
+
+.. image:: images/pipx.png
+
+To permanently modify the default environment variables, follow these steps. Note that to change System variables, you need non-restricted access to your machine (i.e. Administrator rights).
+
+1. Click Start and search for â€˜edit environment variablesâ€™ or open System properties,
+   Advanced system settings.
+2. Click the Environment Variables button.
+3. To change System variables, you need non-restricted access to your machine
+   (i.e. Administrator rights). Add the following paths to your PATH environment variable:
+
+   a. ``%USERPROFILE%\AppData\Roaming\Python\Python37\Scripts``
+   b. ``%USERPROFILE%\.local\bin``
+
+.. image:: images/env-var.png
+
+Open a new command prompt and verify that pipx is available::
+
+    $ pipx --version
+
+You should see a version number after entering in this command, such as: ``0.12.3.1``.
+If you get an error instead, such as ``'pipx' is not recognized as an internal or external command,
+operable program or batch file.``, please check that your environment variables have been updated.
+
+You can now install CumulusCI with::
+
+    $ pipx install cumulusci
+
+Now `verify your installation`_.
+
+
+
+Verify Your Installation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In a new terminal window can verify that CumulusCI is installed correctly by running ``cci version``:
+
+.. code:: console
+
+    $ cci version
+    CumulusCI version: 3.19.0 (/path/to/bin/cci)
+    Python version: 3.8.5 (/path/to/bin/python)
+
+    You have the latest version of CumulusCI.
+
+You can also use this command in the future to check whether your CumulusCI installation is up to date.
+
+Still need help? `CumulusCI's issues on GitHub <https://github.com/SFDO-Tooling/CumulusCI/issues>`_ may have something useful.
+
+
+
 Connect to GitHub
 -----------------
+In order to allow CumlusCI to work with your CumulusCI projects in GitHub, you need to connect GitHub as a service in ``cci``.
+
+First, `create a new personal access token <https://github.com/settings/tokens/new>`_ with both "repo" and "gist" scopes specified.
+(Scopes appear as checkboxes when creating the personal access token in GitHub).
+Copy the access token to use as the password when configuring the GitHub service.
+
+Next, run the following command and provide your GitHub username and the access token as the password::
+
+    $ cci service connect github
+
+You can verify the GitHub service is connected by running ``cci service list``:
+
+.. image:: images/service-list.png
+
+Once you've configured the ``github`` service it will be available to **all** CumulusCI projects.
+Services are stored in the global CumulusCI keychain by default.
+
+
 
-Work on an existing CumulusCI project
+Work on an Existing CumulusCI Project
 -------------------------------------
+Before working on an existing CumulusCI project you need to:
+
+* `Install CumulusCI`_
+* `Install git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+
+
```

Comment:
could use an introduction along the lines of "Use this section if there is an existing CumulusCI project on GitHub that you'd like to work on. Note: CumulusCI does not support projects stored on other git hosts such as BitBucket or GitLab at this time."